frontend/src/component/feature/FeatureStrategy/FeatureStrategyMenu/FeatureStrategyMenu.tsx

```diff
@@ -161,56 +162,81 @@ export const FeatureStrategyMenu = ({
 
     return (
         <StyledStrategyMenu onClick={(event) => event.stopPropagation()}>
-            {displayReleasePlanButton ? (
+            {addConfigurationEnabled ? (
                 <PermissionButton
-                    data-testid='ADD_TEMPLATE_BUTTON'
+                    data-testid='ADD_STRATEGY_BUTTON'
                     permission={CREATE_FEATURE_STRATEGY}
                     projectId={projectId}
                     environmentId={environmentId}
-                    onClick={openReleasePlans}
+                    onClick={openMoreStrategies}
                     aria-labelledby={dialogId}
-                    variant='outlined'
+                    variant={variant}
                     sx={{ minWidth: matchWidth ? '282px' : 'auto' }}
                     disabled={Boolean(disableReason)}
                     tooltipProps={{
                         title: disableReason ? disableReason : undefined,
                     }}
                 >
-                    Use template
+                    Add configuration
```

Comment:
Diff is weird here, but essentially if `addConfigurationEnabled` we show the new button that opens the modal. Otherwise, show the 3 buttons we had previously.